src/ripple/app/ledger/tests/common_ledger.h

```diff
@@ -236,11 +240,23 @@ trust(TestAccount& from, TestAccount const& issuer,
                 std::string const& currency, double amount,
                 Ledger::pointer const& ledger, bool sign = true);
 
+// Simplest form of close_and_advance.
 void
 close_and_advance(Ledger::pointer& ledger, std::shared_ptr<Ledger const>& LCL);
 
-Json::Value findPath(Ledger::pointer ledger, TestAccount const& src, 
-    TestAccount const& dest, std::vector<Currency> srcCurrencies, 
+// Call this function if you want to advance the ledger time.
+void
+close_and_advance(Ledger::pointer& ledger,
+    std::shared_ptr<Ledger const>& LCL, std::uint32_t advSeconds);
+
+// Call this function if you want to retry transactions that fail with
+// a `ter` or `tec`.
+void
+close_and_advance(Ledger::pointer& ledger, std::shared_ptr<Ledger const>& LCL,
+    std::uint32_t advSeconds, std::unique_ptr<LocalTxs>& retries);
```

Comment:
1. I agree that functionality for advancing the ledger should be in `jtx`.  However, `Ledger_test.cpp` still relies on `common_ledger.cpp`.  Once `Ledger_test.cpp` is ported to `jtx`, then it should be possible to salvage the necessary parts out of `common_ledger` and move them to `jtx`.  To me it seems the time for that move is not yet right.
2. For the moment, I believe `LocalTxs` is the only appropriate choice to hold the list of retryables.  The mock relies on the `LocalTxs::sweep()` and `LocalTxs::apply()` interfaces to do its work.  Until those interfaces disappear, due to the Fees work, I believe this is the right choice.